### DISCUSSION TITLE
Новые фичи для трейнспавнера

### DISCUSSION THREAD

**Discussion by cyber-dream** [metrostroi-repo/MetrostroiAddon#197] [Ideas]
Собираю здесь предложения от комьюнити:

- Установка номера вагона вручную
- Возможность сохранить пресет
- кастомные модели
- ориентация вагонов в пространстве
- задавать составность для каждого поддерживаемого количества вагонов

**Comment by MrSandManFM** [comment]
- Пресеты (посоставно\повагонно - нужное подчеркнуть. Частично реализовано)
- Возможность давать игроку выбор из нескольких вариантов в дропдаунах, если скин требует какие-то конкретные (в данный момент можно фиксировать только один вариант в настройках скина)
- Выбор модели салона
- Разная логика ориентации промежутков Е-образных
